src/main/scala/be/doeraene/sjsreflect/Reflect.scala

```diff
@@ -45,6 +45,13 @@ object Reflect {
   def getClassForName(fqcn: String): Option[Class[_]] =
     classesByName.get(fqcn)
 
+  /**
+    * returns an array of all classes specified by the `reflectEnumerateClass()` operator
+    * (and descendants if the  `selectDescendentClasses` selector is used)
+    */
+  def enumerateClasses: js.Array[Class[_]] = classArray
```

Comment:
Same comment as above for the Scaladoc style.

Also, returning the same array is not safe, as arrays are mutable. A client could modify the returned array and pollute everyone else's versions. You should make a copy. Also consider returning a (Scala) `Array[Class[_]]` rather than a `js.Array[_]`, to be more consistent with the rest of the API (notably `getDeclaredConstructors`). So basically:
```scala
def enumerateClasses: Array[Class[_]] = classArray.toArray
```
(since `toArray` makes a copy already)